### PR TITLE
Bugfix: Org settings make public by default toggle

### PR DESCRIPTION
**Pull request by giancorzo**
https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_workitems/edit/30081

### CODE REVIEW

File: source/SIL.AppBuilder.Portal.Frontend/src/data/containers/resources/organization/with-data-actions.tsx [hunk 1]
```diff
@@ -54,17 +54,16 @@ export function withDataActions<T>(WrappedComponent) {
...
-
-      this.forceUpdate();
     }
 
-    updateAttributes = (attributes: OrganizationAttributes) => {
+    updateAttributes = async (attributes: OrganizationAttributes) => {
       const { organization, dataStore } = this.props;
       const { id, type } = organization;
 
-      return dataStore.update(q => q.replaceRecord({
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/data/containers/resources/organization/with-data-actions.tsx]: returning a promise behaves the same her as awaiting it and not returning

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx [hunk 2]
```diff
@@ -46,28 +46,15 @@ export type IProps =
...
 class SettingsRoute extends React.Component<IProps> {
 
-  updateOrganization = async (payload: OrganizationAttributes) => {
+  updateOrganization = (payload: OrganizationAttributes) => {
 
     const { t, updateAttributes } = this.props;
 
     try {
-      await updateAttributes(payload);
+      updateAttributes(payload);
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx]: why was this await removed?

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx [hunk 3]
```diff
@@ -156,8 +143,16 @@ class SettingsRoute extends React.Component<IProps> {
...
-  query(mapRecordsToProps),
+  query(({match: { params: { orgId } } }) => ({
+    cacheKey: [`org-${orgId}`],
+    organization: [q => buildFindRecord(q, TYPE_NAME, orgId), buildOptions({
+      include: ['organization-product-definitions.product-definition', 'groups']
+    })],
+  })),
   withLoader(({ organization }) => !organization),
+  withOrbit(({ organization }) => ({
+    organization: q => q.findRecord({ type: TYPE_NAME,id: organization.id,})
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx]: if findRecord takes an object, you could just do `q.findRecord(organization)` to keep things concise?

Comment (NullVoxPopuli) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx]: also, @giancorzo is this the only place that needs this kind of treatment? or would it be worth it to roll this withOrbit generically into query?

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx (hunk 2 above)
Comment (giancorzo) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx]: I added them back, not sure why is not showing it

**Pull request merged**